src/test/java/cr/pernix/dashboard/resources/ProjectResourceTest.java

```diff
@@ -106,9 +117,23 @@ public void testGetAll() {
         List<Project> projectsList = response.readEntity(new GenericType<List<Project>>() {
         });
         Assert.assertEquals(testProjects.size(), projectsList.size());
-        deleteAll(testProjects);
+        deleteAll(projectsList);
     }
-    
+
+    @Test
+    public void testGetByUser() {
+        List<Project> testProjects = insertTestProjects(5);
+        Assert.assertTrue(testProjects.size() == 5);
+        int userId = testProjects.get(0).getUser().getId();
+        String path = "project/byUser/%d";
```

Comment:
Recently, I found some warnings throwed by sonar qube, related to variable declarations. In this case you can use final String path, because this variable does not change in the current scope. Use "final" in other similar cases.